lib/web_ui/lib/src/engine/canvaskit/text.dart

```diff
@@ -62,32 +62,16 @@ class CkParagraphStyle implements ui.ParagraphStyle {
       skTextStyle.fontSize = fontSize;
     }
 
-    if (fontFamily == null ||
-        !skiaFontCollection.registeredFamilies.contains(fontFamily)) {
-      fontFamily = 'Roboto';
-    }
-    skTextStyle.fontFamilies = [fontFamily];
+    skTextStyle.fontFamilies = _getActualFontFamilies(fontFamily);
```

Comment:
suggestion _getActualFontFamilies -> _getResolvedFontFamilies

Reply:
Named to `_getEffectiveFontFamilies`